lib/login.js

```diff
@@ -33,7 +34,7 @@ module.exports = async function login() {
 
     return mgmtToken;
   } catch (err) {
-    console.log('Failed to login into Contentful');
+    console.log(chalk.red('Failed to login into Contentful.'));
```

Comment:
I'd prefer if we could change this to something like

```suggestion
    console.log(`${chalk.red('error:')} Failed to login into Contentful.'));
```
Output printed entirely in red is very hard to see on terminals with dark backgrounds

Reply:
Good point, I'll do that for all error messages.